doc/Jamfile

```diff
@@ -40,7 +40,7 @@ doxygen reference
 
 actions doxygen-postprocessing
 {
-  python3 $(THIS_PATH)/doxygen_postprocessing.py "$(>)"
+  python $(THIS_PATH)/doxygen_postprocessing.py "$(>)"
```

Comment:
We cannot change this yet, because on some platforms `python` still points to python2.